package.json

```diff
@@ -6,6 +6,9 @@
     "packages/*"
   ],
   "devDependencies": {
+    "@rdfjs/types": "^1.1.0",
```

Comment:
We should already have this one, so can be removed here.

Reply:
Ah yes, in `resolutions`